### PR TITLE
[DOCFIX] Add doc for updateConf command

### PR DESCRIPTION
**Pull request by maobaolong**
Add documentation for updateConf command

### CODE REVIEW

**Review by HelloHorizon — CHANGES_REQUESTED** [review]
Thanks @maobaolong ! Left minor comment

File: docs/en/operation/Admin-CLI.md [hunk 1]
```diff
@@ -300,3 +301,13 @@ $ ./bin/alluxio fsadmin ufs --mode readOnly hdfs://ns
...
+
+### updateConf
+
+The `updateConf` command provides a way to update config for current running services, the request is sent to alluxio master directly,
+but the other services like worker, fuse, s3 proxy or some other clients will aware the config changed and sync the config.
+
+```console
+$ ./bin/alluxio fsadmin updateConf key1=val1 key2=val2
+Updated 2 configs
+```
```
Comment (HelloHorizon) [inline docs/en/operation/Admin-CLI.md:313]: I assume this command don't have general support for dynamic configurations update. Can you also add the supported configuration list in this doc https://github.com/Alluxio/alluxio/pull/13643?

Comment (maobaolong) [reply docs/en/operation/Admin-CLI.md]: Sure

**Comment by maobaolong** [comment]
@HelloHorizon Thanks for the suggestion, I add a chapter, PTAL.

**Review by HelloHorizon — APPROVED** [review]
LGTM

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Comment by HelloHorizon** [comment]
@Xenorith can you cherry-pick this doc fix pr?

**Comment by Xenorith** [comment]
allxuio-bot, cherry-pick this to branch-2.7 please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**